src/lib/Mapper/SearchHitToContentSuggestionMapper.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Search\Mapper;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\Content\Search\SearchHit;
+use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
+use Ibexa\Contracts\Search\Mapper\SearchHitToContentSuggestionMapperInterface;
+use Ibexa\Contracts\Search\Model\Suggestion\ContentSuggestion;
+use Ibexa\Contracts\Search\Provider\ParentLocationProviderInterface;
+
+final class SearchHitToContentSuggestionMapper implements SearchHitToContentSuggestionMapperInterface
+{
+    private ConfigResolverInterface $configResolver;
+
+    private ParentLocationProviderInterface $parentLocationProvider;
+
+    public function __construct(
+        ParentLocationProviderInterface $parentLocationProvider,
+        ConfigResolverInterface $configResolver
+    ) {
+        $this->configResolver = $configResolver;
+        $this->parentLocationProvider = $parentLocationProvider;
+    }
+
+    public function map(SearchHit $searchHit): ?ContentSuggestion
+    {
+        $content = $searchHit->valueObject;
+
+        if (!$content instanceof Content) {
+            return null;
+        }
+
+        $rootLocationId = $this->configResolver->getParameter('content.tree_root.location_id');
+
+        $mainLocation = $content->getVersionInfo()->getContentInfo()->getMainLocation();
+
+        if ($mainLocation === null) {
+            return null;
+        }
+
+        $parentsLocation = $mainLocation->path;
+        $position = array_search((string)$rootLocationId, $parentsLocation);
+        if ($position !== false) {
+            $parentsLocation = array_slice($parentsLocation, (int)$position);
+        }
+
+        $parentCollection = $this->parentLocationProvider->provide($parentsLocation);
+
+        return new ContentSuggestion(
+            $searchHit->score ?? 50,
```

Comment:
technically 50 is an integer, not a float

```suggestion
            $searchHit->score ?? 50.0,
```